modular_citadel/code/modules/mob/living/silicon/robot/dogborg_equipment.dm

```diff
@@ -29,11 +29,19 @@ SLEEPER CODE IS IN game/objects/items/devices/dogborg_sleeper.dm !
 	attack_verb = list("nibbled", "bit", "gnawed", "chomped", "nommed")
 	var/status = 0
 
-/obj/item/dogborg/jaws/attack(atom/A, mob/living/silicon/robot/user)
+/obj/item/dogborg/jaws/small/attack(atom/A, mob/living/silicon/robot/user)
 	..()
+	user.cell.use(500) // Less cell use for the smaller jaws.
 	user.do_attack_animation(A, ATTACK_EFFECT_BITE)
 	log_combat(user, A, "bit")
 
+/obj/item/dogborg/jaws/big/attack(atom/A, mob/living/silicon/robot/user)
+	..()
+	user.cell.use(1000) // Secborg can only harm with the baton, why does the K9 get a free module to do 12 damage with?

```

Comment:
I suggest you code in a new cell_cost variable to /obj/item/dogborg/jaws and replace these code changes with the below one instead, less copypasting.
```
/obj/item/dogborg/jaws/attack(atom/A, mob/living/silicon/robot/user)
	..()
	user.cell.use(cell_cost)
	user.do_attack_animation(A, ATTACK_EFFECT_BITE)
```
Just remember to add the 1000 and 500 values to its two subtypes' cell_cost.

Also remove the `log_combat()` from here, since every item attack() should be logged anyway, this pretty redunant anyway.

Reply:
:ok_hand:

Reply:
:ok_hand: